r2i/tflite/engine.h

```diff
@@ -17,6 +17,8 @@
 #include <memory>
 
 #include <r2i/tflite/model.h>
+#include <r2i/tflite/prediction.h>
+#include <tensorflow/lite/kernels/register.h>
```

Comment:
Are these needed?

Reply:
Yes, for the resolver related types, like `::tflite::ops::builtin::BuiltinOpResolver`